resource/ckb.toml

```diff
@@ -117,8 +117,16 @@ cell_output_cache_size  = 128
 #
 #     ckb cli secp256k1-lock <pubkey>
 #
+# Also, ckb allows the miner to add any data to the cellbase that he has dug out.
+# The data must be A 0x-prefixed hex string. **Please note** that the data cannot be
+# larger than the capacity value of the current cellbase. If it is larger,
```

Comment:
The normal users don't know the capacity value of the cellbase, this data size may be confused for them.

Reply:
1. First of all, it is not a required field here. If you don’t understand it, you will not fill it out.
2. The capacity of cellbase is dynamically changed. It is impossible to determine the size of data by a static value.
3. I considered a lot of processing solutions, and finally decided to write the details in the comment, and verify the value when generating the block template to ensure that the user can find the filling error as soon as possible.

Reply:
I think it's better to truncate the data when it is too long. And add a warning here to tell users that do not rely on the data here, it may be truncated when the block reward capacity is not enough to hold the data.

Reply:
It should never be panic if the data size overflow sometimes, for the user view it's a bug.

Reply:
The current processing will not panic if overflow RPC will return an error when getting the block template.

Reply:
I changed the implementation to forced truncate